enterprise/cloud.oracle/nbproject/project.xml

```diff
@@ -188,6 +188,23 @@
                         <specification-version>1.10</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
+                    <code-name-base>org.openide.filesystems</code-name-base>
```

Comment:
The `cloud.oracle` is using API from dialogs 7.59 - does it require version 7.59?